db/migrations/20230817004323_forbidden_mission_migration.php

```diff
@@ -61,7 +61,7 @@ public function down(): void
         // Delete from items table
         $this->execute("
             DELETE FROM `items`
-                WHERE `item_id` = 131;
+                WHERE `item_id` = 132;
```

Comment:
I think this was supposed to stay 131? Otherwise we're removing the favor item.

Reply:
This is the down() migration so it should be yeah?